Dockerfile.base

```diff
@@ -47,7 +47,21 @@ RUN groupadd --gid "$USER_GID" "$USERNAME" \
     && echo "$USERNAME ALL=(root) NOPASSWD:ALL" > /etc/sudoers.d/"$USERNAME" \
     && chmod 0440 /etc/sudoers.d/"$USERNAME"
 
-WORKDIR /workspaces/art-dash
+# Setting HOME to dev user's and adding their bin to PATH
+ENV HOME /home/"$USERNAME"
+ENV PATH /home/"$USERNAME"/.local/bin:"$PATH"
+
+# Install art-dash and default configs
+COPY conf/krb5-redhat.conf /etc/krb5.conf
+COPY . /tmp/art-dash
+RUN cp /tmp/art-dash/container/doozer-settings.yaml /home/"$USERNAME"/.config/doozer/settings.yaml \
+ && cp /tmp/art-dash/container/elliott-settings.yaml /home/"$USERNAME"/.config/elliott/settings.yaml \
+ && rm -rf /tmp/art-dash
+
+# Only /workspaces has permissions for default user
+WORKDIR /workspaces
+
+USER "$USER_UID"
 # Clone art-tools and install
 RUN git clone https://github.com/openshift-eng/art-tools.git \
```

Comment:
You mentioned "Not deleting art-tools dir post installation, since it won't work if we do" in the commit description however we are still deleting it here.

Reply:
Good catch!

I forgot to push a commit, will update now.

Reply:
Also based on what Luke said, we don't need to change workspaces. I'll push both the fixes together

Reply:
Perfect! lmk on slack